index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-1">
+  <meta name="description" content="IJS Summit 2022 home page">
+  <meta name="Author" content="Hemerson Foreste">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+  <title>International javaScript Summit home page</title>
+</head>
+<body>
+    <main class="main-index">
+    <header>
+        <div class="top-header">
+            <ul class="top-header-links">
+              <li><a class="facebook" href="" title="facebook"><i class="fab fa-facebook-f"></i></a></li>
+              <li><a class="twitter" href="" title="twitter"><i class="fab fa-twitter"></i></a></li>
+              <li>English</li>
+              <li>My Page</li>
+              <li>Logout</li>
+            </ul>
+        </div>
+          
+          <div class="btn-hamburger-menu mobile-version"><i class="fas fa-bars"></i></div>        
+          
+         <nav class="default-navbar">
+          <div class="btn-close mobile-version"><i class="fas fa-times"></i></div> 
+            <div class="logo-container">
+                <a href="index.html" class="logo home" title="home"><img src="images/LogoHemer.png" alt="logo"></a>
+            </div>                          
+          <ul class="menu">
+              <li><a href="index.html" class="home-index menu-items" title="home"><i class="fas fa-home"></i></a></li> 
+              <li><a href="about.html" class="about-index menu-items">About</a></li>
+              <li><a href="" class="program menu-items">Program</a></li>
+              <li><a href="" class="join menu-items">Join</a></li>
+              <li><a href="" class="sponsor menu-items">Sponsor</a></li>
+              <li><a href="" class="news menu-items">News</a></li>
+          </ul>
+          <button class="campaign" type="button">CC Campaign</button>
+         </nav>
+    </header>
+
+    <section id="Index-Introduction">
+      <p class="greetings">"We connect developers!"</p>
+      <h1 class="event-title">INTERNATIONAL JAVASCRIPT SUMMIT 2022</h1>
+      <p class="event-description">The JavaScript world is constantly evolving and there’s something
+           new to learn every day. Join us to be a part of that change and stay on top of the latest trends.
+      </p>
+      <h2 class="event-date">2022.10.13(THU) ~ 16(SUN)</h2>
+      <p class="event-place">@ Caribe Convention Center of Haiti or Online.</p>
+    </section>
+ </main>
+  
+
+  <section id="main-program">
+      <h2 class="program-title">Main Program</h2>
+      <div class="separator"></div>
```

Comment:
- [ ] **[OPTIONAL]** To make your project look more professional 💯, Kindly fix the overlap between the  `class="event-description"` and the `<h2 class="program-title">Main Program</h2>`. See image below

<img width="295" alt="Loom_NUZV2exxof" src="https://user-images.githubusercontent.com/81468675/152626910-ca6599ec-cb41-4a53-9f66-547e2c39e275.png">